checks/hostname.rb

```diff
@@ -32,8 +26,11 @@ def error_exit(message, code)
   exit code
 end
 
+ENV['PATH'] = ENV['PATH'].split(File::PATH_SEPARATOR).concat(['/opt/puppetlabs/bin']).join(File::PATH_SEPARATOR)
+facter_fqdn = `facter fqdn`
+
 # Check that facter actually has a value that matches the hostname.
 # This should always be true for facter >= 1.7
-error_exit(DIFFERENT + BASE, 1) if Facter.value(:fqdn) != `hostname -f`.chomp
+error_exit(DIFFERENT + BASE, 1) if facter_fqdn != `hostname -f`.chomp
```

Comment:
Ah yes, thanks, I reproduced the issue.  I've added a chomp to the line where facter_fqdn's set.